data/gen.go

```diff
@@ -555,10 +573,36 @@ func main() {
 			break
 		}
 	}
+	// Getting move metadata
+	log.Println("Getting move metadata")
+	move_meta_csv := getCsvReader("data/move_meta.csv")
+	records, err = move_meta_csv.ReadAll()
+	if err != nil {
+		panic(err)
+	}
+	for _, v := range records {
+		mid := parseInt(v[0])
+		minHits := parseInt(v[3])
+		maxHits := parseInt(v[4])
+		minTurns := parseInt(v[5])
+		maxTurns := parseInt(v[6])
+		drain := parseInt(v[7])
+		healing := parseInt(v[8])
+		critRate := parseInt(v[9])
+		ailmentChance := parseInt(v[10])
+		flinchChance := parseInt(v[11])
+		statChance := parseInt(v[12])
+		for i, m := range moves {
+			if m.Id != mid {
+				continue
+			}
+			moves[i].Meta = data_move_meta{minHits, maxHits, minTurns, maxTurns, drain, healing, critRate, ailmentChance, flinchChance, statChance}
+		}
+	}
 	output += "var AllMoves = []MoveData{\n"
 	for _, p := range moves {
-		output += fmt.Sprintf("\t{Name: %q, Type: %d, Category: %s,"+
-			" Targets: %d, Priority: %d, Power: %d, Accuracy: %d},\n", p.Name, p.Type, p.DamageClass, p.Targets, p.Priority, p.Power, p.Accuracy)
+		output += fmt.Sprintf("\t{Name: %q, Type: %d, Category: %s, Targets: %d, Priority: %d, Power: %d, Accuracy: %d, metadata: %s},\n",
```

Comment:
There is no need to have a separate struct `MoveMeta` inside of `MoveData`. Just make all the fields of `MoveMeta` fields of `MoveData` instead.

Also, the format string `%#v` outputs the struct as go code.